internal/setup/keyring_test.go

```
@@ -0,0 +1,560 @@
+	data, err := os.ReadFile(path)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	content := string(data)
+	if !strings.Contains(content, "export WK_ACCOUNT=user@example.com") {
+		t.Errorf("missing WK_ACCOUNT line, content:\n%s", content)
+	}
+
+	// Update account.
+	if err := AppendAccountToCredentialsEnv("other@example.com"); err != nil {
```

golangci-lint (2.5.0)

[error] shadow: declaration of "err" shadows declaration at line 265

(govet)